src/Transfer/Sources/NHost.php

```diff
@@ -228,6 +242,42 @@ public function convertAttribute(array $column): Attribute
         }
     }
 
+    /**
+     * Convert Index
+     * 
+     * @param string $table
+     * @return Index|false
+     */
+    public function convertIndex(array $index): Index|false
+    {
+        $pattern = "/CREATE (?<unique>UNIQUE)? INDEX (?<name>\w+) ON (?<table>\w+\.\w+) USING (?<method>\w+) \((?<columns>\w+)\)/";
+
+        if (\preg_match($pattern, $index['indexdef'], $matches)) {
+            // We only support BTree indexes
+            if ($matches['method'] !== 'btree') {
+                $this->logs[Log::ERROR][] = new Log('Skipping index due to unsupported type: ' . $matches['method'] . ' for index: ' . $matches['name'] . '. Transfers only support BTree.', \time());
+
+                return false;
+            }
+
+            $type = "";
+
+            if ($matches['unique'] === 'UNIQUE') {
+                $type = Index::TYPE_UNIQUE;
+            } else {
+                $type = Index::TYPE_FULLTEXT;
+            }
```

Comment:
The default here should be `TYPE_KEY` if neither `UNIQUE` or `FULLTEXT` were included